### PR TITLE
Added Nepali

### PR DESCRIPTION
**Pull request by bluster980**
Added Nepali #76 
**It would be great if you could add hacktoberfest-accepted label.**

### CODE REVIEW

File: src/i18n.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ import contentTr from './locales/tr/content.json';
 import contentSm from './locales/sm/content.json';
 import contentAs from './locales/as/content.json';
 import contentUr from './locales/ur/content.json';
+import contentNp from './locales/np/content.json';
```
Comment (AndrGab) [inline src/i18n.js:34]: the content.json file is empty.

**Review by AndrGab — CHANGES_REQUESTED** [review]

File: src/i18n.js (hunk 1 above)
Comment (bluster980) [reply src/i18n.js:34]: Sorry, That File Was Unsaved, I'm Uploading It Soon.

Comment (bluster980) [reply src/i18n.js:34]: I Have Added The content.json

Comment (AndrGab) [reply src/i18n.js:34]: Nice. Great Job.

**Review by AndrGab — APPROVED** [review]

**Comment by AndrGab** [comment]
Thank you

**Pull request merged**